NSS-Day-Cohort-55/tabloidcli-tuneful-tuis
# finished adding cascade delete

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout team-cascade
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    run sql create script and then create some AuthorTag, BlogTag, and PostTag instances and then delete the tag or thing associated and see if it cascade deletes the join table instance.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
